pyodide_build/pywasmcross.py

```diff
@@ -262,6 +262,12 @@ def handle_command(line, args, dryrun=False):
         # Don't include any system directories
         if arg.startswith("-L/usr"):
             continue
+        for l in args.replace_libs.split(";"):
+            if len(l)>0:
+                (from_lib,to_lib) = l.split("=")
+                if len(from_lib)>0 and arg.startswith("-l"+from_lib):
```

Comment:
I'm not sure it is a good idea to use startswith instead of equals. For example, `libz` and `libzstd` are very different libraries.

Reply:
fair enough will do